bridge/tfchain_bridge/main.go

```diff
@@ -30,36 +30,54 @@ func main() {
 
 	flag.Parse()
 
-	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stdout})
-	zerolog.SetGlobalLevel(zerolog.InfoLevel)
-	if debug {
-		zerolog.SetGlobalLevel(zerolog.DebugLevel)
-		log.Debug().Msg("debug mode enabled")
-	}
+	logger.Init_logger(debug)
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
 	timeout, timeoutCancel := context.WithTimeout(ctx, time.Second*15)
 	defer timeoutCancel()
 
-	br, err := bridge.NewBridge(timeout, bridgeCfg)
+	br, address, err := bridge.NewBridge(timeout, bridgeCfg)
 	if err != nil {
-		panic(err)
+		log.Fatal().
+			Err(err).
+			Str("event_action", "bridge_init_aborted").
+			Str("event_kind", "error").
+			Str("category", "availability").
+			Msg("the bridge instance cannot be started")
 	}
+	source_log_entry := logger.SourceCommonLogEntry{
+		Instance_public_key: address,
+		Bridge_wallet_address: bridgeCfg.StellarBridgeAccount,
+		Stellar_network: bridgeCfg.StellarNetwork,
+		Tfchain_url: bridgeCfg.TfchainURL,
+		}
+
+	log.Logger = log.Logger.With().Interface("source", source_log_entry).Logger()
 
 	sigs := make(chan os.Signal, 1)
 
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
 
 	go func() {
-		log.Info().Msg("awaiting signal")
+		log.Debug().Msg("awaiting signal")
 		<-sigs
-		log.Info().Msg("shutting now")
+		log.Debug().Msg("shutting now")
 		cancel()
 	}()
 
 	if err = br.Start(ctx); err != nil && err != context.Canceled {
-		log.Fatal().Err(err).Msg("exited unexpectedly")
+		log.Fatal().
+			Err(err).
+			Str("event_action", "bridge_unexpectedly_exited").
+			Str("event_kind", "error").
+			Str("category", "availability").
```

Comment:
Shouldn't this be part of the error context somehow? not printed as strings? May be return some sort of a ErrorContext that implements `error` then once received an ErrorContext can be downcasted, then the data extracted from it and printed. 

The values of the category, event-kind, and action can then be pre-defined constants